### PR TITLE
Integration branch for case flags case links

### PR DESCRIPTION
**Pull request by johnbenjamin-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-7241

### Change description ###
Private Law - Integrate ExUI Features into one DEMO environment

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by Mo-Lala84 — COMMENTED** [review]
approved

**Review by tarikdogan-hmcts — DISMISSED** [review]

**Review by RiteshHMCTS — APPROVED** [review]

**Review by Mo-Lala84 — DISMISSED** [review]
approved

File: src/shared/components/search-result/search-result.component.html [hunk 1]
```diff
@@ -12,7 +12,7 @@ <h2 class="heading-h2" id="search-result-heading__text" tabindex="-1">{{ caseSta
 
     <div *ngIf="(hasResults() || hasDrafts())" class="pagination-top"
       attr.aria-label="{{ getTotalResults() }} results have been found" role="status">
-      <span class="text-16" id="search-result-summary__text">Showing 
+      <span class="text-16" id="search-result-summary__text">Showing
```
Comment (andywilkinshmcts) [inline src/shared/components/search-result/search-result.component.html:15]: Might be a problem that the trailing whitespace has been stripped here - could this mess up the layout, or will the fact that the count is in a span mean that it won't be right next to the Showing text - please check

**Review by andywilkinshmcts — COMMENTED** [review]
Should be OK, though please take a look at the possible issue related to stripping a trailing space from the content of an HTML element. In general avoid automatic reformatting of this nature - turn it off in your IDE

**Review by andywilkinshmcts — APPROVED** [review]

**Review by Mo-Lala84 — DISMISSED** [review]
approved

**Review by RiteshHMCTS — DISMISSED** [review]

File: src/shared/components/search-result/search-result.component.html (hunk 1 above)
Comment (johnbenjamin-hmcts) [reply src/shared/components/search-result/search-result.component.html]: @andywilkinshmcts - I have reverted this change and the space is now back. Thank you.

**Review by Kartheek-S — DISMISSED** [review]

**Pull request merged**